setup.cfg

```diff
@@ -19,3 +19,6 @@ exclude = lib/,src/,docs/,bin/
 [isort]
 # NOTE: local imports are handled special (they do not get handled as "tests").
 forced_separate=tests,pytest_django,pytest_django_test
+
+[metadata]
+license_file=LICENSE
```

Comment:
@pombredanne 
Can you comment on https://github.com/pytest-dev/pytest-django/pull/653, please?  Does it make sense to remove the manifest?

Reply:
@blueyed sure thing